### DISCUSSION TITLE
Simple product without variants

### DISCUSSION THREAD

**Discussion by abathellier** [medusajs/medusa#2534] [Q&A] [answered]
I just want to add a simple product without variants. Is it possible and what is the procedure to follow? 

Is it something like that:

1. add general information (there is no price there?)
2. create a single variant without options with a price

**Comment by srindom** [comment, marked as answer]
You cannot have a product without variants - you would have to create a product with an option and a single variant :)

**Reply by abathellier** [reply]
Thank you @srindom. Are there any plans to add this feature? I think not all products have options... The definition of a simple product in the [Woocommerce](https://woocommerce.com/document/managing-products/) documentation is :

> Simple – covers the vast majority of any products you may sell. Simple products are shipped and have no options. For example, a book.

I think this is a common feature of ecommerce stores. Maybe allowing a single variant without options with some docs is a good solution!

**Reply by abathellier** [reply]
I noticed you have such a feature in your current demo admin panel, how did you achieve this?

<img width="1268" alt="Capture d’écran 2022-11-07 à 11 44 08" src="https://user-images.githubusercontent.com/6159227/200293047-7881d226-6bad-469d-be36-709acf803bee.png">

**Reply by voxsar** [reply]
> You cannot have a product without variants - you would have to create a product with an option and a single variant :)

This cannot be accepted as an answer, it a break down of the problem and a workaround

**Comment by VENGENCE7** [comment]
Facing the same problem, a product without variants should be allowed to be added

**Reply by sambitraze** [reply]
@srindom please answer

**Reply by voxsar** [reply]
hello any answers on thjis

**Comment by voxsar** [comment]
Looks like medusajs does not allow a simple product with a simple set price, i have looked at other threads and so now has put answer on this, i am commenting on all of them until i get a clear answer then I will post it as seperate thread so that people can have a easy one place reference for it

But there needs to be confirmation about what i have figured out for my self after 3 days of scrolling through the github discussion forum

**Reply by NicolasGorga** [reply]
hey @voxsar have you found a conclusion to this? I am currently having a lot of doubts regarding how ProductOptions are handled
